.gitlab/source_test/macos.yml

```diff
@@ -27,7 +28,7 @@ tests_macos:
 
 lint_macos:
   stage: source_test
-  rules: !reference [.on_a6]
+  rules: !reference [.on_a7]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What is this change for ?

Reply:
Future-proofing - I don't want us to accidentally stop linting on macOS the day we stop building Agent 6 by default (if that ever happens).